service/auth/auth.go

```diff
@@ -387,20 +458,32 @@
 }
 
 // issueToken issues a JWT-based token
-func (s Service) issueToken(subject string, fullName string, email string, expiry time.Time) (token string, err error) {
+func issueToken(key *ecdsa.PrivateKey, subject string, fullName string, email string, expiry time.Time) (token string, err error) {
 	claims := jwt.NewWithClaims(jwt.SigningMethodES256,
 		&UserClaims{
 			FullName: fullName,
 			EMail:    email,
 			RegisteredClaims: jwt.RegisteredClaims{
 				ExpiresAt: jwt.NewNumericDate(expiry),
-				Issuer:    Issuer,
+				Issuer:    DefaultIssuer,
 				Subject:   subject,
 			}},
 	)
 	claims.Header["kid"] = DefaultKeyID
 
-	token, err = claims.SignedString(s.apiKey)
+	token, err = claims.SignedString(key)
+	return
+}
+
+func issueRefreshToken(key *ecdsa.PrivateKey, subject string) (token string, err error) {
```

Comment:
Another non-expert question: Should something happen with the old token? I guess it is kept as a "reference".

Reply:
Not sure if I understand your question? which old token?